src/terminal/terminal/ansi_escape_codes.h

```diff
@@ -0,0 +1,197 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+
+#ifndef _GUAC_TERMINAL_ANSI_ESCAPE_CODES_H
+#define __GUAC_TERMINAL_ANSI_ESCAPE_CODES_H
```

Comment:
This `#define` needs to match the `#ifndef` above. See:

https://en.wikipedia.org/wiki/Include_guard#Use_of_#include_guards

Also, though you will see the `_` and `__` prefix used elsewhere in older code, that usage is incorrect as such prefixes are reserved for internal use by the language/compiler. Newer code will serve as a better example:

https://github.com/apache/guacamole-server/blob/6f491946408224a231324663938df36c756b09d2/src/terminal/terminal/xparsecolor.h

See also:

https://en.wikipedia.org/wiki/Include_guard#Discussion